README.md

```diff
@@ -151,10 +244,48 @@ selected = True
 A second way is by providing the name of the task, and the path of a .py file containing the selection code:
 
 ```python
-#%selection_script(name=TASK_NAME, path=./SELECTION_CODE_FILE.py)
+#%selection_script(name=TASK_NAME, path=PATH_TO/SELECTION_CODE_FILE.py)
+```
+
+#### 5.5 Adding job fork environment and/or selection script
+
+If the selection scripts and/or the fork environments are the same for all job tasks, we can add them just once using 
+the `job_selection_script` and/or the `job_fork_env` pragmas. 
+
+Usage:
+
+For a job selection script use:
+
+```python
+#%job_selection_script([language=SCRIPT_LANGUAGE], [path=./SELECTION_CODE_FILE.py], [force=on/off])
+```
+
+For a job fork environment use:
+
+```python
+#%job_fork_env([language=SCRIPT_LANGUAGE], [path=./FORK_ENV_FILE.py], [force=on/off])
+```
+
+The `force` parameter says if the pragma has to overwrite the task selection scripts or fork environment already set or not.
+
+#### 5.6 Adding pre and/or post scripts
+
+Sometimes, specified scripts has to be executed before and/or after a particular task. To add those scripts both 
+`pre_script` and `post_script` exist.
```

Comment:
This sentence is not clear for me. To do that, the solution provides pre_script` and `post_script`.

Reply:
So if I understood well your comment, you're suggesting:
"Sometimes, specified scripts has to be executed before and/or after a particular task. To do that, the solution provides `pre_script` and `post_script` pragmas." ?

Reply:
yes